static inline compilation error w/ vulkan.hpp

Getting this compilation error w/ latest hpp generator (as of 0799e35086cb138215772b8bae671e12702f5138):


In file included from ../../src/graphics/tests/common/utils.h:13:
../../third_party/Vulkan-Headers/include/vulkan/vulkan.hpp:5390:55: error: 'static' function 'getDispatchLoaderStatic' declared in header file should be declared 'static inline' [-Werror,-Wunneeded-internal-declaration]
static  ::VULKAN_HPP_NAMESPACE::DispatchLoaderStatic &getDispatchLoaderStatic() {
                                                      ^
1 error generated.


This is NOT a blocking issue for us as I can just add the missing "inline" to the declaration for the time being.

The vk.xml used to generate the vulkan.hpp in the error above is:

[vk.xml.gz](https://github.com/KhronosGroup/Vulkan-Hpp/files/7296424/vk.xml.gz)

Thank you.

--- a/VulkanHppGenerator.cpp
+++ b/VulkanHppGenerator.cpp
@@ -16653,7 +16653,7 @@ int main( int argc, char ** argv )
       }
   extern VULKAN_HPP_STORAGE_API DispatchLoaderDynamic defaultDispatchLoaderDynamic;
 #  else
-  static ::VULKAN_HPP_NAMESPACE::DispatchLoaderStatic & getDispatchLoaderStatic()
+  static inline ::VULKAN_HPP_NAMESPACE::DispatchLoaderStatic & getDispatchLoaderStatic()
   {
     static ::VULKAN_HPP_NAMESPACE::DispatchLoaderStatic dls;
     return dls;

--- a/vulkan/vulkan.hpp
+++ b/vulkan/vulkan.hpp
@@ -5704,7 +5704,7 @@ namespace VULKAN_HPP_NAMESPACE
       }
   extern VULKAN_HPP_STORAGE_API DispatchLoaderDynamic defaultDispatchLoaderDynamic;
 #  else
-  static ::VULKAN_HPP_NAMESPACE::DispatchLoaderStatic & getDispatchLoaderStatic()
+  static inline ::VULKAN_HPP_NAMESPACE::DispatchLoaderStatic & getDispatchLoaderStatic()
   {
     static ::VULKAN_HPP_NAMESPACE::DispatchLoaderStatic dls;
     return dls;

